scripts/build.sh

```diff
@@ -25,9 +25,9 @@ echo "Previous client cleaned."
 
 echo "Build entry point..."
 mkdir -p ./build/server/views
-jade --client --no-debug --hierarchy --out ./build/server ./server
-echo "var jade = require('jade/runtime');module.exports=" | \
-    cat - ./build/server/views/index.js > ./build/server/views/index.js.tmp
+pug --client --no-debug --out ./build/server ./server
```

Comment:
I'm tired :tired_face: I wonder how this would work with globally installed npm packages. Which one is prioritary? If I remember well, the global on is. Which can lead to tons of headaches :)